base/kbd/manifest.x86_64.jsonc

```diff
@@ -0,0 +1,784 @@
+/** Human readable report. This is not consumed by boulder */
+{
+	"manifest-version": "0.1",
+	"packages": {
+		"kbd": {
+			"depends": [
+				"interpreter(/usr/lib/ld-linux-x86-64.so.2(x86_64))",
+				"soname(libc.so.6(x86_64))",
+				"soname(libpam.so.0(x86_64))",
+				"soname(libpam_misc.so.0(x86_64))"
+			],
+			"files": [
+				"/usr/bin/chvt",
+				"/usr/bin/deallocvt",
+				"/usr/bin/dumpkeys",
+				"/usr/bin/fgconsole",
+				"/usr/bin/getkeycodes",
+				"/usr/bin/kbd_mode",
+				"/usr/bin/kbdinfo",
+				"/usr/bin/kbdrate",
+				"/usr/bin/loadkeys",
+				"/usr/bin/loadunimap",
+				"/usr/bin/mapscrn",
+				"/usr/bin/openvt",
+				"/usr/bin/psfaddtable",
+				"/usr/bin/psfgettable",
+				"/usr/bin/psfstriptable",
+				"/usr/bin/psfxtable",
+				"/usr/bin/resizecons",
+				"/usr/bin/setfont",
+				"/usr/bin/setkeycodes",
+				"/usr/bin/setleds",
+				"/usr/bin/setmetamode",
+				"/usr/bin/setvtrgb",
+				"/usr/bin/showconsolefont",
+				"/usr/bin/showkey",
+				"/usr/bin/unicode_start",
+				"/usr/bin/unicode_stop",
+				"/usr/bin/vlock",
```

Comment:
It looks like the vlock command requires a PAM configuration file. Use the following:
```
#%PAM-1.0
auth required pam_unix.so
account required pam_unix.so
password required pam_unix.so
session required pam_unix.so
```
See [openssh](https://github.com/snekpit/main/tree/main/console/openssh) for what to name the pam file in the repo and also where/how to install it correctly.

Reply:
So this would be `pkg/vlock.pam` and would be installed to `/usr/share/defaults/etc/pam.d/vlock`

Reply:
tbh I have no idea what this does, I just wanted to get the package in for the keymaps. Thanks for the hint!